game/src/ecs/components/PositionComponent.java

```diff
@@ -35,6 +38,9 @@ public PositionComponent(@DSLContextMember(name = "entity") Entity entity) {
      * @return the position of the associated entity
      */
     public Point getPosition() {
+        positionCompLogger.log(
+                CustomLogLevel.DEBUG,
+                "Fetching position for entity '" + entity.getClass().getSimpleName() + "'.");
```

Comment:
Wäre es nicht vielleicht sinnvoll, wenn hier zusätzlich zur Information, dass die Position abgefragt wird, auch der Wert geloggt wird? Fürs Debugging wäre das sicherlich hilfreich.
Das gilt dann genau so für die anderen Stellen im Code wo nur "Fetching XY" oder "Checking if XY" ausgegeben wird.

Reply:
Sehr guter Einwand, habe ich in 6850d456aaa56bf5cd9d2c2ab6ac0e83c2db7a04 hinzugefügt

Reply:
👍